AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/ReconcileAndLocalSave/ReconcileAndLocalSaveQueue.swift

```diff
@@ -71,13 +71,15 @@ class ReconcileAndSaveQueue: ReconcileAndSaveOperationQueue {
     }
 
     func addOperation(_ operation: ReconcileAndLocalSaveOperation, modelName: String) {
+
         serialQueue.async {
             var reconcileAndLocalSaveOperationSink: AnyCancellable?
-
             reconcileAndLocalSaveOperationSink = operation.publisher.sink { _ in
-                self.reconcileAndLocalSaveOperationSinks.with { $0.remove(reconcileAndLocalSaveOperationSink) }
-                self.modelReconcileAndSaveOperations[modelName]?[operation.id] = nil
-                self.reconcileAndSaveQueueSubject.send(.operationRemoved(id: operation.id))
+                self.serialQueue.async {
```

Comment:
since this fixes a regression, do you mind putting a paragraph in the PR description explaining why the regression happened and how/why this call fixes it? So we capture it in our history.